cells/_automation/ciJobs.nix

```diff
@@ -0,0 +1,25 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs iohk-nix;
+  inherit (nixpkgs) lib;
+  inherit (inputs.cells.prelude.library) project extra-compilers;
+
+  ciProject = project.appendModule {
+    flake.variants =
+      lib.genAttrs extra-compilers
+      (x: {compiler-nix-name = x;});
+  };
+
+  ciJobs =
+    ciProject.flake'.ciJobs
+    // {
+      devShells = cell.devshells;
+    };
+
+  # path prefixes of jobs that should not be required for CI to pass (end up in `nonrequired`), empty for now:
+  nonRequiredPaths = map lib.hasPrefix [];
+in
+  nixpkgs.callPackages iohk-nix.utils.ciJobsAggregates {inherit ciJobs nonRequiredPaths;}
```

Comment:
is this new? should we be using it in other places?

Reply:
yeah I have been duplicating the logic to a few project, so upstreamed to iohk-nix today.